### DISCUSSION TITLE
Network map manual

### DISCUSSION THREAD

**Discussion by ybizeul** [Koenkk/zigbee2mqtt#5721] [General]
Hello folks,

I'm having a hard time figuring out the network map, I think a documentation would be useful.

![image](https://user-images.githubusercontent.com/182035/104368802-de2f0200-551c-11eb-8463-090ac9f2579f.png)

Consider the following network, and some assumptions :

- Green means End Device
- Blue means Router
- Star is Coordinator
- Link quality is between 0 - 255 (I have 0 here, can't be good I suppose)
- Solid lines are the link to the coordinator

Now... I'm displaying (upper right hand corner) links that "isParent" and "isAChild"

How can I have devices that are not parent or child of anything ? How are they communicating with the network if they're not even linked to the coordinator ?

EDIT : `parents_yann` was indeed electrically turned off, as well as `couloir_1` so that makes sense, but `bureau_switch` and `salon_canape` _seems_ isolated and I don't understand why

**Comment by Koenkk** [comment]
@nurikk could we add some legend in the frontend?

**Comment by ybizeul** [comment]
Here is another example, I know `couloir_1` is off at the switch so that's normal, but I still fail to understand how the group on the left communicates with the network. But it is functional

![image](https://user-images.githubusercontent.com/182035/105036034-35bcf880-5a5c-11eb-9a01-d441ef29861d.png)

**Comment by vincentBesseau** [comment]
Hello all, 

I have the same question about my network 🤔 
I have 7 devices that do not seem to be connected to the network but are fully functional
A device only works one day and after that it never sends information again ("capteur de mouvement 3")

Can you explain me what appened please ?
![image](https://user-images.githubusercontent.com/33568805/200845155-1f09675e-6b4f-49dc-b497-2fb02d3d727c.png)
